test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -339,15 +291,10 @@ instance Arbitrary BlockHeader where
 
 instance Arbitrary SlotId where
     shrink _ = []
-    arbitrary = toEnum <$> choose (0, maxBound `div` 2)
-
-newtype NumberOfSlots = NumberOfSlots Word64 deriving Show
-
-instance Arbitrary NumberOfSlots where
-    shrink (NumberOfSlots 0) = []
-    shrink (NumberOfSlots n) = NumberOfSlots <$> [0, n `div` 2, n - 1]
-    arbitrary = NumberOfSlots . fromIntegral
-        <$> choose (0 :: Int, 4 * (fromIntegral slotsPerEpoch))
+    arbitrary = do
+        ep <- choose (0, 10)
+        sl <- choose (0, 100)
```

Comment:
why not `sl <- choose (0, 21600)`?
Also is it possible to have slot 0? or epoch 0?
It is because it's just an Id I suppose?

Reply:
hm, is it even needed? I mean the whole generator for `SlotId`? The [`Slotting ordering`](https://github.com/input-output-hk/cardano-wallet/pull/126/files#diff-00fd2cdc39324db62230aba60bf1287bR87) test suite ^ is using only some explicit values for SlotId...

Reply:
Not only, there's a first property that compare arbitrary slots against the initial `SlotId 0 0` (making sure they're all greater than it).

Why not `21600` ? Well, to really show that it doesn't matter here and is completely irrelevant for the testing of this module. The only thing we really test is that we compare slot correctly. There's no other assumption about `SlotId` (and that's what we try to preserve in the code as well).